include/deal.II/numerics/data_out_dof_data.h

```diff
@@ -413,6 +413,17 @@ namespace internal
         const UpdateFlags update_flags,
         const bool        use_face_values);
 
+      ParallelDataBase(
+        const unsigned int               n_datasets,
+        const unsigned int               n_subdivisions,
+        const std::vector<unsigned int> &n_postprocessor_outputs,
+        const dealii::hp::MappingCollection<dim, spacedim> &mapping,
+        const std::vector<
+          std::shared_ptr<dealii::hp::FECollection<dim, spacedim>>>
+          &               finite_elements,
+        const UpdateFlags update_flags,
+        const bool        use_face_values);
```

Comment:
Since you only use this new constructor, you no longer need the legacy one in ll. 405-414. This struct is in the `internal` namespace, so you can probably safely delete it without deprecating.

Reply:
This is still needed by the other post processors.

Reply:
Okay, leave it then.